spec/requests/api/v3/npq_participants_spec.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "NPQ Participants API", :with_default_schedules, type: :request, with_feature_flags: { api_v3: "active" } do
+  let(:token) { LeadProviderApiToken.create_with_random_token!(cpd_lead_provider:) }
+  let(:bearer_token) { "Bearer #{token}" }
+  let(:npq_lead_provider) { create(:npq_lead_provider) }
+  let(:cpd_lead_provider) { create(:cpd_lead_provider, npq_lead_provider:) }
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  before { default_headers[:Authorization] = bearer_token }
+
+  describe "GET /api/v3/participants/npq" do
```

Comment:
could you add a spec for the single endpoint as well, also can we add specs for the action endpoints? those can be in a separate request spec for ease maybe?